v3/testdata/build

```diff
@@ -23,391 +23,391 @@ build
 SELECT y FROM a
 ----
 project [out=(1)]
-  columns: a.y:1
-  projections:
-    variable (a.y) [in=(1)]
-  inputs:
-    scan [out=(0,1)]
-      columns: a.x:0 a.y:1
+ |- columns: a.y:1
```

Comment:
Neat. I was actually thinking of something similar recently. Did you consider using the [unicode box drawing characters](https://en.wikipedia.org/wiki/Box-drawing_character)? For example, this could look like:

```
project [out=(1)]
├ columns: a.y:1
├ projections
│ └ variable (a.y) [in=(1)]
└ inputs
    └ scan [out=(0,1)]
        └ columns: a.x:0 a.y:1
```

Reply:
I think the difference in visibility is marginal so I kept it simple.